source/postprocess/geoid.cc

```diff
@@ -934,9 +1071,9 @@ namespace aspect
       {
         prm.enter_subsection("Geoid");
         {
-          prm.declare_entry("Include the contributon from dynamic topography", "true",
+          prm.declare_entry("Include the contributon from topography", "true",
```

Comment:
Oh, changing input parameters will break every parameter file that used this parameter. This is something we try to avoid as much as possible. However, there is a typo in this parameter (contributon), which we have to fix anyway. Could you fix that as well, since you change the line anyway?